portals/application/modules/rest/controllersRaw/PeerServersController.php

```diff
@@ -586,16 +562,6 @@ public function optionsAction()
                     'required' => false,
                     'comment' => '',
                 ),
-                'name' => array(
```

Comment:
You need to apply this changes into rest/controllers/PeerServersController.php as well